src/dvclive/studio.py

```diff
@@ -120,6 +114,8 @@ def post_to_studio(live, event):
             live._studio_events_to_skip.add("data")
             live._studio_events_to_skip.add("done")
     elif event == "data":
+        for path, num_points in live._num_points_read_from_file.items():
+            live._num_points_sent_to_studio[path] = num_points
```

Comment:
Is it enough to move the logic you had before down here (and drop `live._num_points_read_from_file`)?

```python
for name, plot in plots.items():
    live._nb_points_sent_to_studio[path] = len(plot)
```

Reply:
My bad I didn't see I could have access to `plots` here. I though I needed to reload the file, thus my change. I corrected that.